app/src/main/java/org/systers/mentorship/view/fragments/TasksFragment.kt

```diff
@@ -41,12 +44,23 @@ class TasksFragment : BaseFragment() {
 
         rvTasks.apply {
             layoutManager = LinearLayoutManager(context)
-            adapter = TasksAdapter(taskViewModel.taskList, markTask)
+            adapter = TasksAdapter(taskViewModel.taskList, markTask, deleteTask)
         }
 
         ivAddItem.setOnClickListener {
             showDialog()
         }
+
+        taskViewModel.successfulDelete.observe(this, Observer { successful ->
+            // TODO: remove task from list
```

Comment:
is this TODO, still left to do?

Reply:
It isn't. The TasksAdapter class takes care of that. I wrote that to remind myself, and forgot to delete it. Will force-push deleting it now.